Simplified/Reader2/UI/NYPLBaseReaderViewController.swift

```diff
@@ -17,7 +17,7 @@ import R2Shared
 
 
 /// This class is meant to be subclassed by each publication format view controller. It contains the shared behavior, eg. navigation bar toggling.
-class NYPLBaseReaderViewController: UIViewController, NYPLBackgroundWorkOwner, Loggable {
+class NYPLBaseReaderViewController: UIViewController, Loggable {
```

Comment:
do we still use this class anywhere? I'm wondering if we should just delete it.

Reply:
It's still being used in `NYPLReaderReadiumView` which is part of R1.
I will do a clean up and remove anything R1 related once R2 is ready .

Reply:
gotcha, cool